static/app/views/dashboards/manage/tableView/table.tsx

```diff
@@ -0,0 +1,186 @@
+        isLoading={isLoading}
+        isError={false}
+        header={
+          <SavedEntityTable.Header>
+            <SavedEntityTable.HeaderCell data-column="star" />
+            <SavedEntityTable.HeaderCell data-column="name">
+              {t('Name')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="project">
+              {t('Project')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="envs">
+              {t('Envs')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="filter">
+              {t('Filter')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="num-widgets">
+              {t('Widgets')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="created-by">
+              {t('Creator')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="last-viewed">
+              {t('Last Viewed')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="created" noBorder>
+              {t('Date Created')}
+            </SavedEntityTable.HeaderCell>
+            <SavedEntityTable.HeaderCell data-column="actions" />
+          </SavedEntityTable.Header>
+        }
+        isEmpty={dashboards.length === 0}
+        // TODO: DAIN-715 Update empty message
+        emptyMessage={t('No dashboards found')}
+      >
+        {dashboards.map((dashboard, index) => (
+          <SavedEntityTable.Row
+            key={dashboard.id}
+            isFirst={index === 0}
+            data-test-id={`table-row-${index}`}
+          >
+            <SavedEntityTable.Cell hasButton data-column="star">
+              <SavedEntityTable.CellStar
+                isStarred={dashboard.isFavorited ?? false}
+                // TODO: DAIN-718 Add star functionality
+                onClick={() => {}}
+              />
+            </SavedEntityTable.Cell>
+            <SavedEntityTable.Cell data-column="name">
+              <SavedEntityTable.CellName
+                to={`/organizations/${organization.slug}/dashboards/${dashboard.id}`}
+              >
+                {dashboard.title}
+              </SavedEntityTable.CellName>
+            </SavedEntityTable.Cell>
+            <SavedEntityTable.Cell data-column="project">
+              <SavedEntityTable.CellProjects projects={dashboard.projects} />
+            </SavedEntityTable.Cell>
+            <SavedEntityTable.Cell data-column="envs">
+              {/* TODO: DAIN-712 Add environments after they are exposed in the API */}
+              <SavedEntityTable.CellEnvironments environments={[]} />
+            </SavedEntityTable.Cell>
+            {/* TODO: DAIN-716 Add release filter as tokens */}
+            <SavedEntityTable.Cell data-column="filter">{'-'}</SavedEntityTable.Cell>
```

Comment:
I think we usually use ' \u2014 ' for this. Anyways, not really important since it looks like you will be replacing this soon.